From 7c34983e336fb0ae0e9cbb644c4d03005c4d920d Mon Sep 17 00:00:00 2001
From: Andrea Bastoni <andrea.bastoni@tum.de>
Date: Fri, 19 Feb 2021 16:29:21 +0100
Subject: [PATCH 3/4] Hook-in the dtb for the s32v234sbc

---
 arch/arm/dts/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
index caaec36e6d..51eff37f92 100644
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -392,6 +392,7 @@ dtb-$(CONFIG_TARGET_S32G274AEVB) += fsl-s32g274aevb.dtb
 dtb-$(CONFIG_S32G274ARDB) += fsl-s32g274ardb.dtb
 dtb-$(CONFIG_S32G274ARDB2) += fsl-s32g274ardb2.dtb
 dtb-$(CONFIG_TARGET_S32R45EVB) += fsl-s32r45.dtb
+dtb-$(CONFIG_TARGET_MPXS32V234) += fsl-s32v234sbc.dtb
 
 dtb-$(CONFIG_TARGET_DRAGONBOARD410C) += dragonboard410c.dtb
 dtb-$(CONFIG_TARGET_DRAGONBOARD820C) += dragonboard820c.dtb
-- 
2.39.2

